### PR TITLE
Updated read the docs file

### PR DESCRIPTION
**Pull request by JohannesLC**
Tested here to work:
https://fmd-weather.readthedocs.io/en/latest/functionality.html

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Please note that the changes before version 1.10.0 have not been documented.
 
-v3.2.1
```
Comment (wielas) [inline docs/changelog.rst:8]: Why don't we separate the changes? What actually changed between 3.2.1(0) and 3.2.2?

File: .readthedocs.yaml [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
 
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+    # You can also specify other tool versions:
+    # nodejs: "20"
+    # rust: "1.70"
+    # golang: "1.20"
```
Comment (wielas) [inline .readthedocs.yaml:15]: Kind of a noise. These are documentation notes, we don't need them here.

**Review by wielas — CHANGES_REQUESTED** [review]

**Review by wielas — APPROVED** [review]

File: docs/changelog.rst (hunk 1 above)
Comment (JohannesLC) [reply docs/changelog.rst:8]: Agreed, updated

File: .readthedocs.yaml (hunk 2 above)
Comment (JohannesLC) [reply .readthedocs.yaml:15]: It's the standard from ReadTheDocs. Suggest keeping it in

**Pull request merged**